lib/util/cache/package/sqlite.ts

```diff
@@ -0,0 +1,91 @@
+import Sqlite from 'better-sqlite3';
+import type { Database, Statement } from 'better-sqlite3';
+import * as upath from 'upath';
+import { logger } from '../../../logger';
+import { ensureDir } from '../../fs';
+
+export class SqlitePackageCache {
+  private readonly upsertStatement: Statement<unknown[]>;
+  private readonly getStatement: Statement<unknown[]>;
+  private readonly cleanupStatement: Statement<unknown[]>;
+
+  static async init(cacheDir: string): Promise<SqlitePackageCache> {
+    const sqliteDir = upath.join(cacheDir, '/renovate/renovate-cache-sqlite');
```

Comment:
```suggestion
    const sqliteDir = upath.join(cacheDir, 'renovate/renovate-cache-sqlite');
```
otherwise it will store on `/renovate/renovate-cache-sqlite`

Reply:
> otherwise it will store on `/renovate/renovate-cache-sqlite`

Do you have reproduction? Because it's not what I experienced in my local setup...